React/tic-tac/src/app/components/Header/styles.scss

```diff
@@ -1,51 +1,27 @@
+@import url('https://fonts.googleapis.com/css?family=Pacifico');
+
 body {
   margin: 0;
 }
 
-.app {
-  text-align: center;
-}
-
 .app-logo {
   animation: app-logo-spin infinite 20s linear;
-  height: 80px;
+  height: 100px;
 }
 
 .app-header {
   background-color: #EEE;
-  color: #333;
-  height: 150px;
+  height: auto;
   padding: 20px;
   text-align: center;
 }
 
 .app-title {
-  font-size: 1.5em;
-}
-
-.app-intro {
-  font-size: large;
-}
-
-ol,
-ul {
-  padding-left: 30px;
-}
-
-.kbd-navigation .square:focus {
-  background: #DDD;
-}
-
-.game {
-  display: flex;
-  flex-direction: row;
-}
-
-.gameInfo {
-  margin-left: 20px;
+  color: #333;
+  font-family: 'Pacifico', cursive;
```

Comment:
Move the font definitions to their own file on the sass variables folder. This way you can replace the whole app font editing only one central file. 😬